project/Dependencies.scala

```diff
@@ -37,19 +37,21 @@ object Dependencies {
   val scalaTestVersion = "3.1.4"
   val scalaCheckVersion = "1.15.1"
 
+    def getScalaVersion() = {
```

Comment:
```suggestion
  def getScalaVersion() = {
```

Reply:
the formatting is done by `scalafmtSbt`, should I change it?

Reply:
are you sure? `sbt scalafmtSbt` seems to change it for me as well

Reply:
I'll re-run it so 👍

Reply:
`scalafmt` was breaking, fixed, and re-formatted